src/assets/js/react/components/Template/TemplateActivateButton.js

```diff
@@ -44,7 +44,6 @@ export class TemplateActivateButton extends React.Component {
    * @since 4.1
    */
   static propTypes = {
-    template: PropTypes.object,
```

Comment:
The `template` prop is still being passed to the component, so wouldn't this propType still apply?